ironmon_tracker/screens/TrackerScreen.lua

```diff
@@ -830,8 +834,11 @@ function TrackerScreen.drawMovesArea(pokemon, opposingPokemon)
 		end
 
 		-- If move info is randomized and the user doesn't want to know about it, hide it
+		-- If fighting a ghost, hide effectiveness
 		local showEffectiveness = true
-		if not Options["Reveal info if randomized"] then
+		if Battle.isGhost then
+			showEffectiveness = false
```

Comment:
Same as previous functionality, just additional setting it to false no matter what against a Ghost